WooCommerce/Classes/ViewRelated/Orders/Collect Payments/CollectOrderPaymentUseCase.swift

```diff
@@ -596,6 +596,11 @@ private extension CollectOrderPaymentUseCase {
     }
 
     func presentBackedReceiptFailedNotice(with error: Error?, onCompleted: @escaping (() -> Void)) {
+        // TODO: consider removing this under #12864, when we have some other way to handle notices.
```

Comment:
Thanks for adding this note, I added a subtask to https://github.com/woocommerce/woocommerce-ios/issues/12864#issuecomment-2141259637.